packages/@orchidui-icons/public/icons/add-image.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="orchid" fill="currentColor" viewBox="0 0 25 25">
+  <g >
+    <g >
+      <path fill="currentColor" fill-rule="evenodd" d="M20.76 3.144a2 2 0 0 1 1.994 1.85l.006.15v5a1 1 0 0 1-2 0v-5h-16v10.1l4.994-4.995a1.25 1.25 0 0 1 1.667-.09l.101.09 2.945 2.945a1 1 0 0 1 .293.707c0 .891-1.077 1.337-1.707.707l-2.415-2.414-5.657 5.657a1.002 1.002 0 0 1-.221.167v1.126h9a1 1 0 0 1 0 2h-9a2 2 0 0 1-1.995-1.85l-.005-.15v-14a2 2 0 0 1 1.85-1.995l.15-.005h16Zm-4.5 4a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm2.5 6a1 1 0 0 1 1 1v2h2a1 1 0 1 1 0 2h-2v2a1 1 0 1 1-2 0v-2h-2a1 1 0 1 1 0-2h2v-2a1 1 0 0 1 1-1Z"  clip-rule="evenodd"/>
```

Comment:
Great when making the whole SVG to inherit the normal CSS color: of the surrounding element 👍